Dockerfile.pulumi

```diff
@@ -0,0 +1,64 @@
+FROM python:3.7-slim-buster as pants-base
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install --yes \
+    build-essential \
+    curl \
+    unzip \
+    wait-for-it
+
+RUN adduser \
+        --disabled-password \
+        --gecos '' \
+        --home /home/grapl \
+        --shell /bin/bash \
+        grapl
+
+USER grapl
+ENV USER=grapl
+WORKDIR /home/grapl
+
+RUN curl -O https://get.pulumi.com/releases/sdk/pulumi-v2.21.2-linux-x64.tar.gz && \
+    tar -xzvf pulumi-*.tar.gz && \
+    mv pulumi pulumi-bin
+ENV PATH=/home/grapl/pulumi-bin:${PATH}
```

Comment:
Any particular reason we don't use a Pulumi image from DockerHub like pulumi/pulumi:v2.21.2 ?

Reply:
Since installing Pulumi is the least of what this Dockerfile is doing, I figured it was more important to align with our other Dockerfile.